plugin/versioning/versioning.cc

```diff
@@ -30,15 +30,15 @@ static int forced_versioning_init(void *p __attribute__ ((unused)))
 
   DBUG_ENTER("forced_versioning_init");
   vers_force= true;
```

Comment:
It's possible to load a plugin through [queries](https://mariadb.com/kb/en/mariadb/plugin-commands/)
And it looks like a data race when someone `CREATE TABLE` while plugin is loading.